src/Android/Avalonia.Android/AvaloniaView.cs

```diff
@@ -27,6 +27,8 @@ public AvaloniaView(Context context) : base(context)
             AddView(_view.View);
             _root = new EmbeddableControlRoot(_view);
             _root.Prepare();
+            if (AndroidPlatform.UseDeferredRendering)
```

Comment:
I don't think this is the right place for starting the renderer. On desktop it's done form platform-independent classes.

Reply:
You are also aren't stopping the renderer, so it gets GC-rooted from the render loop.

Reply:
That's also why popups doesn't work, they aren't using AvaloniaView, since AvaloniaView is just a wrapper on top of our toplevel.